agent/connect/ca/provider_consul.go

```diff
@@ -478,7 +479,7 @@ func (c *ConsulProvider) SignIntermediate(csr *x509.CertificateRequest) (string,
 			x509.KeyUsageDigitalSignature,
 		IsCA:           true,
 		MaxPathLenZero: true,
-		NotAfter:       effectiveNow.AddDate(1, 0, 0),
+		NotAfter:       effectiveNow.Add(c.config.IntermediateCertTTL + extra),
```

Comment:
Yeah I can do that. I made that change because when testing with very short TTLs it is strange to have to use `65s` instead of `5s` because otherwise the cert is expired immediately. But as a user I too would be surprised to see it is valid 6months + 1minute.